lib/setup_logging.dart

```diff
@@ -31,35 +36,14 @@ Future<void> setupLogging() async {
   final startupLogger = Logger("Startup");
   startupLogger.info("App starting, logging initialized.");
 
-  final packageInfo = await PackageInfo.fromPlatform();
-  final deviceInfo = DeviceInfoPlugin();
+  final metadata = MetaData();
+  await metadata.init();
```

Comment:
if `MetaData` a singledton or not? in `finamp_logs_helper.dart` you used `GetIt` to get the instance...